### PR TITLE
Decline terms and conditions page

### PR DESCRIPTION
**Pull request by tahsinaislam**

### CODE REVIEW

File: app/controllers/state_file/questions/declined_terms_and_conditions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module StateFile
+  module Questions
+    class DeclinedTermsAndConditionsController < QuestionsController
+
+      def self.show?(_intake)
+        false
+      end
```
Comment (bengolder) [inline app/controllers/state_file/questions/declined_terms_and_conditions_controller.rb:7]: I think we should be storing their answer and whether they consented ([similar to GYR](https://github.com/codeforamerica/vita-min/blob/main/app/models/intake.rb#L380)) and that this should be shown if they have any answer other than "yes"

File: app/views/state_file/questions/terms_and_conditions/edit.html.erb [hunk 2]
```diff
@@ -9,10 +9,17 @@
...
+        <%= f.button :submit, name: "#{f.object_name}[consented_to_terms_and_conditions]", value: 'yes', class: 'button button--primary' do %>
+          <%= t(".accept") %>
+        <% end %>
+        <%= f.button :submit, name: "#{f.object_name}[consented_to_terms_and_conditions]", value: 'no', class: 'button' do %>
+          <%= t(".decline") %>
+        <% end %>
+      </div>
+    </div>
+  <%end %>
+<% end %>
```
Comment (tahsinaislam) [inline app/views/state_file/questions/terms_and_conditions/edit.html.erb:25]: I couldn't find simpler versions of the yes no buttons. It looks like every example in the codebase is similar to this format if I'm not mistaken.

Comment (bengolder) [reply app/views/state_file/questions/terms_and_conditions/edit.html.erb:25]: You're right! I can't find better examples either. This seems like a good opportunity for a refactor at some point!

**Review by bengolder — APPROVED** [review]
Approved! Great work! Thank you Tahsina!

**Pull request merged**